js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogInsightsLineChart.tsx

```diff
@@ -195,6 +207,41 @@ export const AssetCatalogInsightsLineChart = React.memo(
       }),
       [renderTooltipFn, rgbColors],
     );
+
+    const chartRef = useRef(null);
+    const onClick = (event: any) => {
```

Comment:
Is there a better type than `any` available here?

Reply:
I updated this event type to be a `React.MouseEvent`, though the downstream `getElementsAtEventForMode` function expects a regular `Event`. Is there a way of casting a `React.MouseEvent` to be an `Event`? 

I ended up doing `event as unknown as Event` because `event as Event` was throwing an error, but not sure if there's a better way of handling this.